addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1061,6 +1061,98 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
     },
 });
 
+const ImagepickerUserValueWidget = UserValueWidget.extend({
+    tagName: 'we-imagepicker',
+    events: {
+        'click .o_we_edit_image': '_onEditImage',
+        'click .o_we_remove_image': '_onRemoveImage',
+    },
+
+    /**
+     * @override
+     */
+    start: async function () {
+        await this._super(...arguments);
+        const allowedSelector = this.el.dataset.allowVideos;
+        this.allowVideos = allowedSelector ? this.$target.is(allowedSelector) : false;
+
+        this.editImageButton = document.createElement('we-button');
+        this.editImageButton.classList.add('o_we_edit_image', 'fa', 'fa-fw', 'fa-edit');
+
+        this.removeImageButton = document.createElement('we-button');
+        this.removeImageButton.classList.add('o_we_remove_image', 'fa', 'fa-fw', 'fa-times');
+        this.removeImageButton.title = _t("Remove");
+
+        return this.containerEl.append(this.editImageButton, this.removeImageButton);
```

Comment:
`append` is an experimental feature which would crash on edge, use `appendChild`. Also don't return the result even if it does not change anything.

Reply:
Edge has supported `append` for a while, it's only IE that doesn't support it. But it is true that it's still not in the standard so I'll change that regardless.